include/functions

```diff
@@ -2305,7 +2305,8 @@
                     # Check for additional config files included as follows
                     # "include sites-enabled/*.conf"
                     elif [ $(echo ${VALUE} | grep -F -c "*.conf") -gt 0 ]; then
-                        for FOUND_CONF in $(ls ${CONFIG_FILE%nginx.conf}${VALUE%;*}); do
+                        if [ ${VALUE:0:1} != '/' ]; then VALUE=${CONFIG_FILE%nginx.conf}; fi
```

Comment:
This looks like it is not POSIX compatible. Can you rewrite it using `if [ "$(echo ${VALUE} | ${CUTBINARY} -c1)" != "/" ]` and test it that works correctly on your system?

Reply:
Totally, I was afraid it might be the case but then lazy tested it in dash and thought it would be ok. I have to check for some option to check for non-posix constructions automatically.

Anyway rewritten and tested on 100+ systems, everything looks ok (in our environment).